__tests__/app.test.js

```diff
@@ -133,6 +133,15 @@ describe("GET /api", () => {
                     expect(body.msg).toBe("Bad request");
                 })
             })
+            test("200: Should retrieve an array of all article objects with the necessary properties (now including COMMENT COUNT AS IF WE DIDN'T ALREADY WRITE FOR IT!)", () => {
```

Comment:
??? description?

Reply:
I was being ridiculous. It's been amended.